src/universal/routes/logout.js

```diff
@@ -1,7 +1,16 @@
-export default {
+import {makeWhitelistedReducer} from 'universal/redux/makeReducer';
+
+const reducerWhitelist = [
+  'notifications'
+];
+
+export default (store) => ({
   path: '/logout',
   getComponent: async(location, cb) => {
     const component = await System.import('universal/containers/Logout/LogoutContainer');
+    const newReducer = makeWhitelistedReducer(reducerWhitelist);
+    store.replaceReducer(newReducer);
```

Comment:
Sadly, this doesn't work (yet). Here's what's going on.

1. Navigate to `/me`, then hit gear on UserHub to signout. You'll get to `/me/settings`.
2. Hit Sign Out
3. Router routers to `/logout`

...then `MeSettings.js` re-renders one last time as it unmounts and, guess what?, userSettings reducer is no longer there! Which causes a case of the ickys.

See possible fix in comment above...